src/codechandler/CodecHandler.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Copyright (C) 2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "CodecHandler.h"
+
+namespace
+{
+constexpr const char* netflixFramerateUuid = "NetflixFrameRate";
```

Comment:
sorry for the boredom but I edited my comment too late 
constexpr vars name must be uppercase like old "static const"
NETFLIX_FRAMERATE_UUID

Reply:
No need for apologies, thanks for your patience with me!